src/pipecat/services/aws.py

```diff
@@ -33,6 +33,40 @@
     raise Exception(f"Missing module: {e}")
 
 
+def language_to_aws_language(language: Language) -> str | None:
+    language_map = {
+        Language.CA: "ca-ES",
+        Language.ZH: "cmn-CN",
+        Language.DA: "da-DK",
+        Language.NL: "nl-NL",
+        Language.NL_BE: "nl-BE",
+        Language.EN: "en-US",
+        Language.EN_US: "en-US",
+        Language.EN_AU: "en-AU",
+        Language.EN_GB: "en-GB",
+        Language.EN_NZ: "en-NZ",
+        Language.EN_IN: "en-IN",
+        Language.FI: "fi-FI",
+        Language.FR: "fr-FR",
+        Language.FR_CA: "fr-CA",
+        Language.DE: "de-DE",
+        Language.HI: "hi-IN",
+        Language.IT: "it-IT",
+        Language.JA: "ja-JP",
+        Language.KO: "ko-KR",
+        Language.NO: "nb-NO",
+        Language.PL: "pl-PL",
+        Language.PT: "pt-PT",
+        Language.PT_BR: "pt-BR",
+        Language.RO: "ro-RO",
+        Language.RU: "ru-RU",
+        Language.ES: "es-ES",
+        Language.SV: "sv-SE",
+        Language.TR: "tr-TR",
+    }
+    return language_map.get(language, "en-US")
```

Comment:
Do we need to return a default value here? `self._settings["language"]` already has logic to default to `en-US`.